include/nodejs/node_osrm_support.hpp

```diff
@@ -1229,6 +1229,24 @@ argumentsToTableParameter(const Nan::FunctionCallbackInfo<v8::Value> &args,
         }
     }
 
+    if (obj->Has(Nan::New("scale_factor").ToLocalChecked()))
+    {
+        auto scale_factor = obj->Get(Nan::New("scale_factor").ToLocalChecked());
+
+        if (!scale_factor->IsNumber())
+        {
+            Nan::ThrowError("scale_factor must be a number");
+            return table_parameters_ptr();
+        }
+        else if (scale_factor->NumberValue() < 1)
```

Comment:
Same again - docs say > 0, code enforces >= 1 (not the same as above btw, which does > 1, not >=).

Should allow any number I think, no need to restrict this, it doesn't trigger any computation growth if it's large.